src/components/EventList.tsx

```diff
@@ -35,66 +35,85 @@ const EventList = ({ eventResponses, events, onEventResponse, org } : EventListP
-                                <FormattedDate
-                                    day="2-digit"
-                                    month="long"
-                                    value={ Date.parse(e.start_time) }
-                                />
-                                , <FormattedTime
-                                    value={ Date.parse(e.start_time) }
-                                />
-                            </View>
-                            <View data-test="end-time">
-                                <FormattedDate
-                                    day="2-digit"
-                                    month="long"
-                                    value={ Date.parse(e.end_time) }
-                                />
-                                , <FormattedTime
-                                    value={ Date.parse(e.end_time) }
-                                />
-                            </View>
-                            <View data-test="location-title">{ e.location.title }</View>
-                            { response ? (
-                                <Button
-                                    data-test="event-response-button"
-                                    marginTop="size-50"
-                                    onPress={ () => onEventResponse(e.id, org.id, true) }
-                                    variant="cta">
-                                    <Msg id="misc.eventList.undoSignup"/>
-                                </Button>
-                            ) : (
-                                <Button
-                                    data-test="event-response-button"
-                                    marginTop="size-50"
-                                    onPress={ () => onEventResponse(e.id, org.id, false) }
-                                    variant="cta">
-                                    <Msg id="misc.eventList.signup"/>
-                                </Button>
-                            ) }
-                            <NextLink href={ `/o/${org.id}/events/${e.id}` }>
-                                <a>
-                                    <Button marginTop="size-50" variant="cta">
-                                        <Msg id="misc.eventList.moreInfo"/>
-                                    </Button>
-                                </a>
-                            </NextLink>
-                        </Flex>
+                { events?.map((event) => {
+                    const response = eventResponses?.find(response => response.action_id === event.id);
+                    return (<EventListItem
+                        key={ event.id }
+                        event={ event }
+                        onEventResponse={ onEventResponse }
+                        org={ org }
+                        response={ response }
+                    />
                     );
-                }) }
+                })
+                }
```

Comment:
I think these should be on the same line. It always looks weird when two braces are on the same indentation but different lines. But even if it didn't look weird, this is an unnecessary change to the code (compare how it looks on `main`, i.e. the left side of the GitHub diff).

I'm surprised the linter had no opinion about this. 😊